eudico/fxmodules/consensus.go

```diff
@@ -14,22 +14,35 @@ import (
 type ConsensusAlgorithm int
 
 const (
-	ExpectedConsensus ConsensusAlgorithm = iota
+	none ConsensusAlgorithm = iota
+	ExpectedConsensus
 	MirConsensus
 	TSPoWConsensus
 )
 
+// InjectedConsensusAlgorithm is an ugly hack to replace the deprecated
+// build.Consensus constant, which was used as throughout the code in conditional
+// expressions that execute depending on its value. Ideally, we would be not depend
```

Comment:
typo?
```suggestion
// expressions that execute depending on its value. Ideally, we should not depend
```

Also, I agree, I would rather not depend on a global variable, but I think that we can leave it for now.  Once we add support for spawning several subnet stacks from the same peer we may find a better way to deal with it.